app/assets/javascripts/src/Metamaps.Visualize.js

```diff
@@ -143,6 +143,9 @@ Metamaps.Visualize = {
       self.mGraph.graph.empty()
     }
 
+
+    if (self.type == 'ForceDirected' && Metamaps.Active.Mapper) $.post('/maps/' + Metamaps.Active.Map.id + '/events/user_presence')
```

Comment:
No, it doesn't make that check too many places. The better check might actually be for Metamaps.Active.Map which *should* never be set if there is no active map. And then same with Metamaps.Active.Topic